specification/containerservices/resource-manager/Microsoft.ContainerService/2017-08-31/managedClusters.json

```diff
@@ -154,6 +154,58 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/accessProfiles/{roleName}": {
+      "get": {
+        "tags": [
+          "ManagedClusters"
+        ],
+        "operationId": "ManagedClustersAccessProfiles_Get",
```

Comment:
I think this `operationId` should be `ManagedClusters_GetAccessProfiles` instead. Otherwise the code that gets generated creates a new operations class, and we're forced to pass in `resource_type` for most operations to distinguish whether we're asking for ManagedClusters or ManagedClustersAccessProfiles. See https://github.com/Azure/azure-sdk-for-python/compare/master...mboersma:aks?expand=1 for an example of the generated code.